### ISSUE TITLE
Latest serve-d (nightly) does not work on Ubuntu 22.04 LTS

### ISSUE THREAD

**Issue by dejlek** [Pure-D/serve-d#280]
I finally had some spare time to look into why the D extension did not work as expected on Ubuntu (luckily I do not use Ubuntu so often so it was not a big deal for me). It appears that pre-compiled serve-d binary that the D extension is downloading does not work on the mentioned Ubuntu version. (https://github.com/Pure-D/serve-d/releases/download/nightly/serve-d_linux-nightly-x86_64-20220322-855a87.tar.xz)

        dejan@dejan-lp:~/.local/share/code-d/bin 
        » ./serve-d 
        object.Exception@../../../.dub/packages/requests-1.1.7/requests/source/requests/ssl_adapter.d(193): loading openssl: unknown version for init
        Segmentation fault (core dumped)

So, basically serve-d does not work in VS Code out of box. - It needed a little bit of push, so I cloned serve-d, built the executable myself, and placed it where the extension expected it, and changed the configuration (to "frozen") so the extension does not overwrite the good serve-d with the latest pre-compiled one. And voila, it all works again.

**Comment by WebFreak001** [comment]
I'm not sure why you think I'm inactive :p

I'll investigate this, I thought this would have been solved by dependency upgrades, so I assumed the current nightly (and pre-release) worked, as they also work fine on archlinux.

**Comment by EsoCoding** [comment]
Thanx, i used a binary from 2 day old nightbuild, replaced it. And its finally running now. Almost needed to use a crappy ide without fully darkmode. Thanks i can finally start learning D now 👍😎

**Comment by AssertionBit** [comment]
Same issue was found on ArchoLinux. OpenSSL version == `3.1.1`. I can't change the version of package because it's mostly used in all packages that used in runtime of other packages, so serve-d are not working at all at this time.

**Comment by WebFreak001** [comment]
Are you using the serve-d nightly release? It should already work again

**Comment by AssertionBit** [comment]
Yes, only nightly release

**Comment by dejlek** [comment]
I did not enounter this error in the past few months, so you may close it as far as I am concerned.

**Comment by Speykious** [comment]
It stopped working again. I'm on Arch Linux with OpenSSL 3.2.0, using nightly release.

```
object.Exception@../../../.dub/packages/requests/2.1.1/requests/source/requests/ssl_adapter.d(204): loading openssl: unknown version Tuple!(int, "major", int, "minor")(3, 2) for init
```

**Comment by WebFreak001** [comment]
upstream: https://github.com/ikod/dlang-requests/issues/161
